pkgs/applications/editors/xmlcopyeditor/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, pkgs, aspell, boost, expat, expect, intltool, libxml, libxslt, pcre, xercesc, wxwidgets }:
+
+stdenv.mkDerivation rec {
+  name = "xmlcopyeditor-${version}";
+  version = "1.2.1.3";
+
+  src = fetchurl {
+    name = "${name}.tar.gz";
+    url = "https://sourceforge.net/projects/xml-copy-editor/files/xmlcopyeditor-linux/${version}/${name}.tar.gz/download";
```

Comment:
Use the mirror url instead: `mirror://sourceforge/project/xml-copy-editor/xmlcopyeditor-linux/${version}/${name}.tar.gz`. Then the name will also be set to the last part automatically, so you can remove your name attribute.

Reply:
I think you cand do even shorter: `mirror://sourceforge/xml-copy-editor/${name}.tar.gz`

Reply:
It does indeed!